docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3934 @@
+* Adds the ability to display the latest agent version in agent explorer {kibana-pull}153643[#153643]
+* Adds table tabs showing summary of metrics {kibana-pull}153044[#153044]
+* Adds warning to Edit Rule Flyout when publicUrl is not configured {kibana-pull}149832[#149832]
+
+Dashboard::
+Adds the ability to scroll the panel section {kibana-pull}145628[#145628]
+
+Elastic Security::
+For the Elastic Security 8.8.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.8.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Adds audit logging for core CRUD operations {kibana-pull}152118[#152118]
+* Adds modal to display versions changelog {kibana-pull}152082[#152082]
+
+Infrastructure::
+* Adds the logs tab to the Hosts View {kibana-pull}152995[#152995]
+* Adds Alerts tab into Hosts View {kibana-pull}149579[#149579]
+* Adds refactoring to the Time and Position log stream state {kibana-pull}149052[#149052]
+
+Machine Learning::
+* Adds support for custom URLs in jobs for Data Frame Analytics {kibana-pull}154287[#154287]
+* Adds support to filter fields from grouping in Explain Log Rate Spikes{kibana-pull}153864[#153864]
+* Adds log pattern analysis in Discover {kibana-pull}153449[#153449]
+
+Management::
+Adds support for global settings {kibana-pull}148975[#148975]
+
+Maps::
+Adds map.emsUrl to docker env variables {kibana-pull}153441[#153441]
+
+Observability::
+* Adds the ability to changes all SLO assets to managed, and indices to hidden {kibana-pull}154953[#154953]
+* Adds Exploratory View to a separate app {kibana-pull}153852[#153852]
+
+Platform::
+Adds text {kibana-pull}151631[#151631]
+
+Security::
+* Adds CloudFormation agent install method {kibana-pull}155045[#155045]
+* Adds Vul mgmt flyout details panel {kibana-pull}154873[#154873]
+* Adds Vulnerabilities Table {kibana-pull}154388[#154388]
+* Adds the ability to select a theme preference for {kib} in the User Profile {kibana-pull}151507[#151507]
+
+Uptime::
+Adds UUID to RuleAction {kibana-pull}148038[#148038]
+
+For more information about the features introduced in 8.8.0, refer to <<whats-new,What's new in 8.8>>.
+
+[[enhancements-and-bug-fixes-v8.8.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.8.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.8.0]]
+=== Enhancements
+Alerting::
+* Adds the ability to predefine IDs when you create connectors {kibana-pull}155392[#155392]
+* Adds the ability to allow the footer added to emails sent from {kib} alerting rules to **NOT** be added {kibana-pull}154919[#154919]
+* Adds conditional actions UI for timeframe {kibana-pull}153944[#153944]
+* Adds the ability to filter user activities with pagination {kibana-pull}152702[#152702]
```

Comment:
This should be in the `features` category. The PR had the wrong labels. I updated them accordingly.

Reply:
We should add this PR in the enhancements section https://github.com/elastic/kibana/pull/155151. Sorry for the wrong labels.